pkg/kubectl/cmd/update.go

```diff
@@ -160,3 +176,76 @@ func updateWithPatch(cmd *cobra.Command, args []string, f *cmdutil.Factory, patc
 	_, err = helper.Patch(namespace, name, api.StrategicMergePatchType, []byte(patch))
 	return name, err
 }
+
+func forceUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string, filenames util.StringList) error {
+	schema, err := f.Validator()
+	if err != nil {
+		return err
+	}
+
+	cmdNamespace, err := f.DefaultNamespace()
+	if err != nil {
+		return err
+	}
+
+	mapper, typer := f.Object()
+	r := resource.NewBuilder(mapper, typer, f.ClientMapperForCommand()).
+		ContinueOnError().
+		NamespaceParam(cmdNamespace).DefaultNamespace().
+		FilenameParam(filenames...).
+		ResourceTypeOrNameArgs(false, args...).RequireObject(false).
+		Flatten().
+		Do()
+	err = r.Err()
+	if err != nil {
+		return err
+	}
+	//Update will create a resource if it doesn't exist already, so ignore not found error
+	ignoreNotFound := true
+	// By default use a reaper to delete all related resources.
+	if cmdutil.GetFlagBool(cmd, "cascade") {
+		glog.Warningf("\"cascade\" is set, kubectl will cascade the deletion of the resources managed by this resource (e.g. Pods created by a ReplicationController). Consider using \"kubectl rolling-update\" if you really plan to update a ReplicationController together with its Pods.")
```

Comment:
Wording nits for simplicity/clarity:
- s/will cascade the deletion of the/will delete and recreate all
- s/if you really plan/if you want